lib/vector-core/src/event/metadata.rs

```diff
@@ -123,6 +131,17 @@ impl EventMetadata {
     pub fn set_splunk_hec_token(&mut self, secret: Arc<str>) {
         self.secrets.insert(SPLUNK_HEC_TOKEN, secret);
     }
+
+    /// Adds the value to the dropped fields list, returning the
+    /// position the value will be found in the list.
```

Comment:
Is this comment correct? The function doesn't have a return.

Reply:
Good catch!